ark/util/serialize.ts

```diff
@@ -99,9 +95,9 @@ const _serialize = (
 		case "symbol":
 			return printableOpts.onSymbol(data as symbol)
 		case "bigint":
-			return `${data}n`
```

Comment:
As-is, this would affect `printable` too which I use in a lot of error messages (I'd expect some `arktype` tests would fail?).

Let's add an `opts.onBigint`, then when we call the top-level `snapshot`, we can pass these representations to options but leave the defaults as-is.

Reply:
ok, added! Did you want me to use that somewhere or just add the option?